docs/src/userguide/anatomy/graph.rst

```diff
@@ -1,9 +1,19 @@
 Graph
 =====
 
-In **streaming** mode, Klio makes use of `Google Pub/Sub`_ and `GCS buckets`_ to create a directed acyclic graph (DAG) to string job dependencies together, allowing various modes of execution.
 
-Klio support two modes of execution: :ref:`top-down <top-down>` and :ref:`bottom-up <bottom-up>`.
+Management of a graph of Klio jobs is not supported in **batch** mode.
+We recommend making use of an external orchestration framework (such as `Luigi`_)
+to manage dependencies between Klio batch jobs. In addition, you may find using your job's
+:ref:`event outputs <io-outputs>` helpful.
+
+From this point on, any reference to a Klio job on this page will mean a Klio **streaming** job.
```

Comment:
I would put this line and the paragraph above it in an "attention" admonition, like:

```rst
.. attention::

    Management of a graph ...

    From this point on, ...
```

Just so that it's more obvious to the reader (like myself who skims docs a lot 😅 ).

Reply:
Ah, good call!